python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
I guess I'm missing something. Doing:
```python
f = DiracFile(namePattern='/some/lfn/data.txt', localDir='/home/person/my_analysis/data')
f.get()
```
should do a single copy of `/some/lfn/data.txt` to `/home/person/my_analysis/data/data.txt`. There's no multiple copying involved.